README.rst

```diff
@@ -63,6 +63,10 @@ Configuring the Host Cloud
    happen with the baremetal instances booted from an empty image it speeds
    things up a bit.
 
+#. ``/etc/nova/nova.conf`` needs ``allow_duplicate_networks=true`` in the
```

Comment:
Ah, that makes sense.  I've tested in OSP 7, but it was before the single bmc changes that would make this necessary.